src/pages/Admin.tsx

```diff
@@ -0,0 +1,155 @@
+import { IonButton, IonCol, IonContent, IonGrid, IonHeader, IonIcon, IonInput, IonItem, IonLabel, IonList, IonListHeader, IonPage, IonRadio, IonRadioGroup, IonRow, IonTitle, IonToolbar} from '@ionic/react';
+import { closeCircleSharp } from 'ionicons/icons';
+import './Admin.css';
+
+const Admin: React.FC = () => {
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonTitle>Admin</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent fullscreen>
+        <IonHeader collapse='condense'>
+          <IonToolbar>
+            <IonTitle size="large">Admin</IonTitle>
+          </IonToolbar>
+        </IonHeader>
+        <IonList lines="none">
+            <IonListHeader>
+              <IonLabel>User Settings</IonLabel>
+            </IonListHeader>
+            <IonRadioGroup value={"User Sertings"}>
+              <IonItem>
+                <IonLabel>User Settings 1</IonLabel>
+                <IonRadio slot="start" value = "user_settings_1"></IonRadio>
+              </IonItem>
+
+              <IonItem>
+                <IonLabel>User Settings 2</IonLabel>
+                <IonRadio slot="start" value = "user_settings_2"></IonRadio>
+              </IonItem>
+            </IonRadioGroup>
+            <IonRow>
+
+                <IonItem>
+                  <IonLabel>Maxiumum Team Size</IonLabel>
+                  <IonInput type="number"></IonInput>
+                </IonItem>
+
+                <IonButton size="small">Announcments</IonButton>
+
+                <IonButton size="small">Generate Report</IonButton>
+            </IonRow>
+          </IonList>
+          <IonGrid>
+            <IonRow class="header-row">
+              <IonCol size="1">Name</IonCol>
+
+              <IonCol size="2">Team</IonCol>
+
+              <IonCol size="2">Email</IonCol>
+
+              <IonCol size="2">Total Steps</IonCol>
+
+              <IonCol size="3">Actions</IonCol>
+            </IonRow>
+
+            <IonRow>
```

Comment:
Would it be possible to turn this into more of a looping scenario? Like create a list of all of your fake data objects and then say something like, "for every object in list, display another row"? I think we're going to need it when we start pulling data from an actual database. You should check out Nathan's Manual Step Recording PR for an example of using functions in views.